tools/marvin/marvin/lib/base.py

```diff
@@ -2459,13 +2460,31 @@ def create(cls, apiclient, cluster, services, zoneid=None, podid=None, hyperviso
                   GetDetailExceptionInfo(e)
             return FAILED
 
+    @staticmethod
+    def _check_resource_state(apiclient, hostid, resourcestate):
+        hosts = Host.list(apiclient, id=hostid, listall=True)
+
+        validationresult = validateList(hosts)
+
+        if validationresult[0] == FAIL:
```

Comment:
I hear you, @jburwell, but the API description points out that the return type is a List with three items (and what each item indicates). We can certainly validate return types, but it adds overhead (both in terms of execution time and extra logic clouding things up) for limited value here (it will just throw an exception if the value happened to be None).

Thoughts on that?